telegram/constants.py

```diff
@@ -901,3 +901,18 @@ class WebhookLimit(IntEnum):
     """:obj:`int`: Minimum length of the secret token."""
     MAX_SECRET_TOKEN_LENGTH = 256
     """:obj:`int`: Maximum length of the secret token."""
+
+
+class ChatLimits(IntEnum):
```

Comment:
IIRC we named verything else in singular, i.e. `class <…>Limit(enum)`

Reply:
You appear to be right